frontend/public/components/resource-dropdown.tsx

```diff
@@ -98,9 +148,76 @@ const ResourceListDropdown_: React.SFC<ResourceListDropdownProps> = (props) => {
     ))
     .toArray();
 
+  const recentSearches: JSX.Element[] =
+    !_.isEmpty(recentSelectedList(recentSelected)) &&
+    recentSelectedList(recentSelected)
+      .splice(0, RECENT_SEARCH_ITEMS)
+      .map((modelRef: K8sResourceKindReference) => {
+        const model: K8sKind = resources.find((m) => referenceForModel(m) === modelRef);
+        if (model) {
+          return (
+            <SelectOptionDeprecated
+              key={modelRef}
+              value={modelRef}
+              data-filter-text={`${model.abbr}${model.labelKey ? t(model.labelKey) : model.kind}`}
+            >
+              <span className="co-resource-item">
+                <span className="co-resource-icon--fixed-width">
+                  <ResourceIcon kind={modelRef} />
+                </span>
+                <span className="co-resource-item__resource-name">
+                  <span>
+                    {model.labelKey ? t(model.labelKey) : model.kind}
+                    {model.badge && model.badge === 'Tech Preview' && (
+                      <span className="co-resource-item__tech-dev-preview">
+                        {t('public~Tech Preview')}
+                      </span>
+                    )}
+                  </span>
+                  {isDup(model.kind) && (
+                    <div className="co-resource-item__resource-api text-muted co-truncate co-nowrap small">
+                      {model.apiGroup || 'core'}/{model.apiVersion}
+                    </div>
+                  )}
+                </span>
+              </span>
+            </SelectOptionDeprecated>
+          );
+        }
+        return null;
+      })
+      .filter((item) => item !== null);
+
+  const renderedOptions = () => {
+    const options: JSX.Element[] = [];
+    if (!_.isEmpty(recentSelectedList(recentSelected)) && !!recentList) {
+      options.push(
+        <CloseButton
```

Comment:
Better to add a tooltip on CloseButton. `Clear resent used history` or something like it. Because in first impression it feels like it will remove the all dropdown options below but it will only remove the resent used options. cc: @jerolimov